Monika After Story/game/script-topics.rpy

```diff
@@ -8437,3 +8457,24 @@ label monika_bad_facts_4:
     m 2dfd "[player], if you ever go out into a place where you might need to rely on such a cheap trick, please bring a compass."
     m 2dkc "I would hate for something to happen to you, especially because of false information like this..."
     return
+    
+    init 5 python:
+	addEvent(Event(persistent.event_database,eventlabel="monika_friendship",category=['monika','friends'],prompt="Friendship with literature club members",random=True))
+
+label monika_friendship:
+    m 1euc "You know, even after it all..."
+    m 1ruc "I still miss them."
```

Comment:
1ruc is not a currently valid expression. 

This also raises concerns of what Monika thinks of the other club members. There are some who argue that she does miss them and regret what she did and another half that argues that Monika thinks the other club members were never real and don't matter in the slightest. 

In monika_tsundere, Monika says Natsuki doesn't matter since she doesn't even exist. 
In monika_route, she says that the club didn't matter as soon as she found out it wasn't real. 
In monika_othergames, she says that Yuri's death didn't matter. 
In monika_impression, she makes fun of the other club members as she does impressions of them. 

With all that in mind, this topic currently contradicts a lot of what Monika thinks of the other club members and as such, might not fit well into MAS.